dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -449,9 +450,21 @@ public void publish(List<String> args) {
             publishHelp();
         } else {
             String first = reqVal(args, "--entry");
-            String entryname = optVal(args, "--entryname", null);
             final boolean unpublishRequest = args.contains("--unpub");
-            handlePublishUnpublish(first, entryname, unpublishRequest);
+
+            // --newEntryName is the desired parameter flag, but maintaining backwards compatibility for --entryname
+            String newEntryName = optVal(args, "--newEntryName", null);
+            if (newEntryName == null) {
+                newEntryName = optVal(args, "--entryname", null);
```

Comment:
1. Perhaps a warning that `--entryName` is deprecated? 
2. Also, checking for it here, you're going to be supporting --entryName for workflows. I thought the idea was tool would support --entryName and --newEntryName, because tool already has --entryName, but since it's being added to workflow for the first time, we don't have to worry about backwards compatibility.

I don't feel too strongly about either of these.

Reply:
1. Adding a deprecation warning
2. My initial thinking was that it would seem more consistent to have `--entryname` compatible, but deprecated, for both tools and workflows. Since this is currently (happy to change it) being handled within the AbstractEntryClient, I opted to avoid specifying tools/workflows specifically. Thoughts?